includes/class-edd-html-elements.php

```diff
@@ -496,7 +496,7 @@ public function category_dropdown( $name = 'edd_categories', $selected = 0 ) {
 	 *
 	 * @return string $output Year dropdown
 	 */
-	public function year_dropdown( $name = 'year', $selected = 0, $years_before = 5, $years_after = 0 ) {
+	public function year_dropdown( $form = 'form', $name = 'year', $selected = 0, $years_before = 5, $years_after = 0 ) {
```

Comment:
The new parameter needs to be added to the end. I know it won't be nicely organized, but the order is important for how the function gets used and we need to assume that third party plugins are using this function and we can't break their implementation.

Also, what about just making this an `$id` attribute and letting the user set the exact ID they're looking for? As opposed to building it from a form "name" of sorts. I don't think `$form` is used anywhere else so just letting us set the exact ID we want seems a bit simpler. To try to make it unique for anyone not using the parameter, the default could be `edd-year-dropdown` or something. :woman_shrugging: